src/mockData.js

```diff
@@ -0,0 +1,24 @@
+import Terms from 'components/terms/index'
+import Topics from 'components/topics/index'
+import Questions from 'components/questions/index'
+
+const mockTerms = Terms.DataStore.mockData
+const mockTopics = Topics.DataStore.mockData
+const mockQuestions = Questions.DataStore.mockData
+
+const mock = {
+  terms: {
+    getAll: jest.fn(() => Promise.resolve(mockTerms)),
+    items: mockTerms
+  },
+  topics: {
+    getAll: jest.fn(() => Promise.resolve(mockTopics)),
+    items: mockTerms
+  },
+  questions: {
+    getAll: jest.fn(() => Promise.resolve(mockQuestions)),
+    items: mockQuestions
+  }
+}
```

Comment:
This is our `mockData` helper. It will allow us to grab mock data for each of our modules, including mocking the requests. We can use the mock function to check for calls. They automatically get cleared after each test with code in our `setupTests.js` file, see below.

We will likely be adding more mock requests to this file later on.